### PR TITLE
add image repository example

### PR DESCRIPTION
**Pull request by xcwisc**

### CODE REVIEW

**Comment by blackburn32** [comment]
Looks good, but you should add a root level README that spells out what it is + how it's laid out. See the other extensions in the repo for examples.

File: icon-repository-dragndrop/package.zip
Comment (Sheyne) [inline icon-repository-dragndrop/package.zip:1]: You should not commit a binary file.

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,7 @@
 **/shapelibraries/*.lcsz
 **/*.credentials.local
 **/manifest.local.json
-**/data-connector/bin
\ No newline at end of file
+**/data-connector/bin
+
+# whitelist
+!icon-repository-dragndrop/package.zip
```
Comment (Sheyne) [inline .gitignore]: It seems like committing the package.zip was intentional. What is the goal here? Can we use GitHub releases to do the same thing without mucking up the repo with binary blobs?

Comment (xcwisc) [reply .gitignore]: Yeah I think that's a good idea

**Pull request merged**